### DISCUSSION TITLE
GitHub Skyline doesn't work

### DISCUSSION THREAD

**Discussion by DeeJayLSP** [community/community#66776] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

It has been like this since a few days for me:

![image](https://github.com/community/community/assets/60024671/fdf5398c-b45e-4bec-9d54-7ac55f43157a)

Trying to access a url with someone's username returns a 404:

![image](https://github.com/community/community/assets/60024671/2b85248a-ed4d-4080-9ac7-b5ccb5853ca1)

**Comment by vigo** [comment]
it's not working

**Reply by Pasan-Pahasara** [reply]
yes

**Comment by jm-janzen** [comment]
Yep. No matter what name I enter, I also get the "We did not find any ..." message. Disappointing. :/

**Comment by weather-tracker** [comment]
still not working

**Comment by Pasan-Pahasara** [comment]
still not working

**Comment by SoftwareWajih** [comment]
Still not working Github Skyline is not finding the username.

**Comment by Synex93** [comment]
still not working

**Comment by mironovmeow** [comment]
Still not working

---

Here older dicsusion (month ago) with the same problem: https://github.com/orgs/community/discussions/64942

**Comment by brayden-scott-FHR** [comment]
https://github.com/orgs/community/discussions/64942

In the older discussion, it looks like someone raised a ticket yesterday. Let's hope they get on this soon
